### PR TITLE
Fixed a valgrind issue with the DAG/EdgeFactory

### PR DESCRIPTION
**Pull request by klingaard**

### CODE REVIEW

File: sparta/example/CoreModel/CMakeLists.txt [hunk 1]
```diff
@@ -27,6 +27,7 @@ sparta_application(sparta_core_example)
 # Add the custom regress/regress_valgrind targets.
 add_custom_target (core_example_regress)
 add_custom_target (core_example_regress_valgrind)
+set (VALGRIND_TEST_LABEL valgrind_test)
```
Comment (jackjchen) [inline sparta/example/CoreModel/CMakeLists.txt:30]: NBD, but If it works for you, could we just set the label once up in the common example/CMakeLists.txt

Comment (klingaard) [reply sparta/example/CoreModel/CMakeLists.txt]: Yep

**Review by knute-sifive — APPROVED** [review]

**Comment by jackjchen** [comment]
@knute-sifive I'll look at the CI step for example dir. I think with the bare ctest command it's trying to run valgrind by default.

File: sparta/example/CoreModel/CMakeLists.txt (hunk 1 above)
Comment (klingaard) [reply sparta/example/CoreModel/CMakeLists.txt]: Done.

**Comment by klingaard** [comment]
closes #290

**Review by jackjchen — APPROVED** [review]
Looks good, thanks!

**Pull request merged**